_posts/2018-09-17-pulp-community-health-audit.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Pulp Community Health Audit Report
+author: Brian Bouterse
+---
+In August 2018 the [Open Source and Standards Group](http://community.redhat.com/) completed their
+first Pulp Community Health Audit. This annual audit is a chance to assess the health and diversity
+of the Pulp community, identify growth areas, apply best practices, and check-in on progress made
+towards previous goals.
+
+This blog gives a summary, but you can read the full report
+[here](/files/Community_Audit_Report_Pulp_2018.pdf).
+
+
+# Overview
+
+Overall the Pulp community is healthy, but it
+can improve by making most of its resources more clearly discoverable, e.g. quick start guides, etc.
+Here are the recommended actions for the Pulp community to do to improve itself from the audit:
```

Comment:
Here are the recommended actions from the report for how the Pulp community can improve: